purchase_warning_minimum_amount/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: for your information, the `# -*- coding: utf-8 -*-` is not more required in python 3